gulpfile.js/index.js

```diff
@@ -1,12 +1,16 @@
 /**
- * Every JavaScript file in the `tasks/` directory will be exported with its
- * basename used as the key. These will then be read by Gulp as individual task
- * definitions.
- *
- * This means running `npx gulp hello-world` will run the task exported by
- * `tasks/hello-world.js` if it exists.
+ * Exposes each Gulp task from other files to Gulp
  *
  * @see https://gulpjs.com/docs/en/getting-started/javascript-and-gulpfiles#splitting-a-gulpfile
  */
 
-module.exports = require('require-dir')('./tasks');
+module.exports = {
+  watchPreprocess: require('./tasks/watch-preprocess'),
+  theoToMDX: require('./tasks/theo-to-mdx'),
+  svgToTwig: require('./tasks/svg-to-twig'),
+  build: require('./tasks/build'),
+  buildSass: require('./tasks/build-sass'),
+  buildJS: require('./tasks/build-js'),
+  buildTypes: require('./tasks/build-types'),
+  clean: require('./tasks/clean'),
+};
```

Comment:
Why?

Reply:
- It may make sense for a single file to export multiple tightly-related tasks
- Static require's are more explicit and less magic
- Static require's will very easily be translated into `import`s once we make that transition